apps/src/templates/teacherNavigation/EmptyStateUtils.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import {NavLink, generatePath} from 'react-router-dom';
+
+import {LinkButton} from '@cdo/apps/componentLibrary/button';
+import emptyDesk from '@cdo/apps/templates/teacherDashboard/images/empty_desk.svg';
+import blankScreen from '@cdo/apps/templates/teacherDashboard/images/no_curriculum_assigned.svg';
+import NoUnitAssigned from '@cdo/apps/templates/teacherNavigation/images/NoUnitAssigned.svg';
+import TeacherDashboardEmptyState from '@cdo/apps/templates/teacherNavigation/images/TeacherDashboardEmptyState.svg';
+import i18n from '@cdo/locale';
+
+import {
+  LABELED_TEACHER_NAVIGATION_PATHS,
+  TEACHER_NAVIGATION_PATHS,
+} from './TeacherNavigationPaths';
+
+import styles from './teacher-navigation.module.scss';
+
+export const getNoStudentsEmptyState = () => {
```

Comment:
Totally fine either way, but was curious why you made a new Utils file instead of putting the functions in EmptyState

Reply:
Honestly, `EmptyState` was getting long, ugly, and hard to navigate through with so many functions. It was hard to actually "see" the component in the mess.